### PR TITLE
Gidk 7 guideline

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/audit-logs/audit-logs.component.scss [hunk 1]
```diff
@@ -3,8 +3,8 @@
     text-align: center;
     justify-content: center;
 }
-@media(min-width:1366px){
-    .audit-log-sidebar{
-         max-width: 350px;
-    }
-}
+// @media(min-width:1366px){
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/audit-logs/audit-logs.component.scss:6]: ignore for now

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html [hunk 2]
```diff
@@ -1,5 +1,4 @@
-<h1 class="page-title" *ngIf="(this.auditLogs$ | async).length > 0"> Audit Logs</h1>
-<br />
+<!-- <h1 class="page-title" *ngIf="(this.auditLogs$ | async).length > 0"> Audit Logs</h1> -->
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html:1]: Ignore for now

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/inventory/inventory.component.html [hunk 3]
```diff
@@ -9,7 +9,9 @@
                     <inventory-sidebar *ngIf="!isMobileScreen || (isMobileScreen && activeView !== 'group')"></inventory-sidebar>
                     <!--/sidebar-->
 
-                    <section class="rightBar" *ngIf="!isMobileScreen || (isMobileScreen && activeView === 'group')">
+                    <section class="right-bar" *ngIf="!isMobileScreen || (isMobileScreen && activeView === 'group')"
+                    [ngClass]="{'walcome-inventory-page':!activeView}"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/inventory.component.html:13]: correct spelling of walcome

File: apps/web-giddh/src/app/shared/header/components/aside-help-support/aside-help-support.component.scss [hunk 4]
```diff
@@ -72,12 +71,12 @@
 
                 p{
                     font-size: 14px;
-                    font-family: 'opensans-semibold'; 
+                    font-family: 'opensans-semibold';
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/components/aside-help-support/aside-help-support.component.scss:74]: Please use font variable

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/header/components/aside-help-support/aside-help-support.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/components/aside-help-support/aside-help-support.component.scss:74]: Font variable is not working. It takes time for R&D about font variable

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/inventory/inventory.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/inventory/inventory.component.html]: Done

**Pull request merged**